### PR TITLE
Fix linux2 homework

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 1]
```diff
@@ -1,16 +1,41 @@
 # Linux - Homework 
 
 ### linux2.txt
+Please save your answers in linux2.txt file.
 
-In this exercise you will use also these commands:
+In this exercise, you will use also these commands:
+tree, cp, mv, head, tail, diff, clear, man, cat and, wc.
```
Comment (sleviim) [inline docs/course_materials/exercises/01_Linux/homework_2.md:7]: tree
does this one should be removed?

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 2]
```diff
@@ -1,16 +1,41 @@
...
- **create all directories in one line command**, look in the man or use --help to investigate command options)
+ 1. Create a new directory named **hw2** in your home folder
 
- 2. List in a tree form all the directories in your home directory (don't forget to use man)
- 
- 3. Move welcome.txt to ~/hw1/other
+ 2. Under the directory **hw2** create 3 directories **songs**, **drafts** and **movies**
+  **create all directories in one line command**, look in the man or use --help to investigate command options)
+
+ 3. List in a tree format all the directories in your home directory
```
Comment (sleviim) [inline docs/course_materials/exercises/01_Linux/homework_2.md:15]: same - tree related

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 3]
```diff
@@ -1,16 +1,41 @@
...
+
+ 5. In the file **my_song.txt**, add the lyrics of a song you love.
+
+ 6. Print the first 4 lines of the song.
+
+ 7. Print the last 2 lines of the song.
+
+ 8. Copy the file **my_song.txt** to **new_song.txt** under ~/hw2/songs.
+
+ 9. Write your name in the first and last line of the file **new_song.txt**.
```
Comment (sleviim) [inline docs/course_materials/exercises/01_Linux/homework_2.md:27]: line > lines

File: docs/course_materials/exercises/01_Linux/homework_2.md [hunk 4]
```diff
@@ -1,16 +1,41 @@
...
+
+ 10. Use the command diff and check the difference between the files **my_song.txt** and **new_song.txt**. (don't forget to use man).
+
+ 11. Move the file  **new_song.txt** to ~/hw2/drafts.
+
+ 12. List in a tree format all the directories in your home directory.
```
Comment (sleviim) [inline docs/course_materials/exercises/01_Linux/homework_2.md:33]: same - tree

File: docs/course_materials/exercises/01_Linux/homework_2.md (hunk 1 above)
Comment (shiramax) [reply docs/course_materials/exercises/01_Linux/homework_2.md:7]: nope, Ilana added the command

File: docs/course_materials/exercises/01_Linux/homework_2.md (hunk 2 above)
Comment (shiramax) [reply docs/course_materials/exercises/01_Linux/homework_2.md:15]: Ilana added the command

File: docs/course_materials/exercises/01_Linux/homework_2.md (hunk 3 above)
Comment (shiramax) [reply docs/course_materials/exercises/01_Linux/homework_2.md:27]: done

File: docs/course_materials/exercises/01_Linux/homework_2.md (hunk 4 above)
Comment (shiramax) [reply docs/course_materials/exercises/01_Linux/homework_2.md:33]: we have this command now

**Review by sleviim — APPROVED** [review]

**Pull request merged**